model/event/subscriptions.go

```diff
@@ -311,10 +297,7 @@ func validatePositiveFloat(s string) error {
 }
 
 func (s *Subscription) String() string {
-	id := "???"
-	if s.ID.Valid() {
-		id = s.ID.Hex()
-	}
+	id := s.ID
```

Comment:
Can you preserve the ??? and only set it to s.ID if len(s.ID) == 0?

Reply:
done